src/index.js

```diff
@@ -1,128 +1,93 @@
-import _ from 'lodash';
+import { navBar } from './navBar';
+import { footer } from './footer';
+import { home } from './home';
+import { menu } from './menu';
+import { about } from './about';
+import { contact } from './contact';
```

Comment:
Then update this accordingly.
```suggestion
import navBar from './navBar';
import footer from './footer';
import home from './home';
import menu from './menu';
import about from './about';
import contact from './contact';
```

This way, you could even call your imports any name you prefer, e.g. `import ContactMe from './contact';` would still work fine.